README.md

```diff
@@ -6,6 +6,10 @@ _Write once, run on any cloud ☁️_
 [![godoc](https://godoc.org/github.com/google/go-cloud?status.svg)][godoc]
 [![Coverage](https://codecov.io/gh/google/go-cloud/branch/master/graph/badge.svg)](https://codecov.io/gh/google/go-cloud)
 
+<p align="center">
```

Comment:
I think I'm more okay with it not being centered.

Reply:
Can you say why? I think it looks significantly better when centered

Reply:
It was a grouping thing: having it be with the heading and left-aligned would more strongly associate the logo and heading. But since we're moving the logo to a position that causes a natural break anyway, centering establishes it as a separate element.

For sake of using standard markup, let's use `style` instead:

```suggestion
<p style="text-align:center;">
```